connect/eaas/runner/helpers.py

```diff
@@ -86,3 +95,40 @@ def to_ordinal(val):
     if val > 14:
         return f"{val}{ORDINAL_SUFFIX.get(int(str(val)[-1]), 'th')}"
     return f"{val}{ORDINAL_SUFFIX.get(val, 'th')}"
+
+
+def get_connect_version():
+    env = get_environment()
+    url = f'https://{env["api_address"]}/public/v1'
+    api_key = 'ApiKey MODULE:{}'.format(env['api_key'])
```

Comment:
api_key from environment is already of the form `ApiKey ...`
so ConnectClient(env['api_key'], endpoint=url) is enough